stdlib/public/runtime/Reflection.mm

```diff
@@ -24,7 +24,10 @@
 #include <cstring>
 #include <new>
 #include <string>
+#include <tuple>
+#if !defined(_MSC_VER)
 #include <dlfcn.h>
+#endif
```

Comment:
This include does not seem to be used on all platforms, could you try removing it?